similarity/knn.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package similarity;
+
+message Organizations {
+  message Organization {
+    string name = 1;
+    message Neighbor {
+      string name = 2;
+    }
+    repeated Neighbor neighbors = 3; 
+  }
+  repeated Organization orgs = 4;
+}
```

Comment:
Field numbers correspond to the message they're defined in, and should start from 1, e.g.

```suggestion
message Organizations {
  message Organization {
    string name = 1;
    message Neighbor {
      string name = 1;
    }
    repeated Neighbor neighbors = 2; 
  }
  repeated Organization orgs = 1;
}
```

Neighbor.name is not field of Organization, but of the different message Orginization.Neighbor, etc.